demisto_sdk/tests/integration_tests/pack_validation_test.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
no need for new file, move it to `demisto-sdk/demisto_sdk/tests/integration_tests/validate_integration_test.py`